apps/meteor/server/models/raw/LivechatRooms.ts

```diff
@@ -2264,6 +2264,17 @@ export class LivechatRoomsRaw extends BaseRaw<IOmnichannelRoom> implements ILive
 		);
 	}
 
+	countOpenByAgent(userId: string, extraQuery: Filter<IOmnichannelRoom> = {}) {
```

Comment:
extraQuery is optional on model but required here 🤔 

(Can you update `findOpenByAgent` too? :please: )